button-inc/emissions-elt-demo
# test: add terratest and testing for metabase_vm

Addresses #221. Depends on #219. Adds a testing framework for Terraform.

## Changes

- added Terratest and required frameworks/packages
- added a test for the `metabase_vm` module

## To test

In your console
1. Run `asdf install` to get `go`
2. Run `go mod tidy`
3. Run `export GOOGLE_PROJECT=emissions-elt-demo`
4. Run `go test terraform/modules/metabase_vm/metabase_vm_test.go -v -timeout 10m`
  - Test can take a few minutes, but should end with a Pass
